inlong-sort/sort-connectors/postgres-cdc/src/main/java/org/apache/inlong/sort/cdc/postgres/table/PostgreSQLReadableMetaData.java

```diff
@@ -473,21 +482,50 @@ public static Map<String, Integer> getSqlType(@Nullable TableChange tableSchema)
         return postgresType;
     }
 
+    /**
+     * convert canal operation to canal-json operation type
+     *
+     * @param record
```

Comment:
Please add more desc for `@param` and `@return` tags, otherwise, there are some warnings in the IDE.